js/treeData.js

```diff
@@ -0,0 +1,724 @@
+// Hero Tree Example data
+const introData = {
+  "text": "With Hierplane, you can explore linguistic structures naturally. Try it for yourself!",
+  "nodeTypeToStyle": {
+    "verb": ["color1", "strong"],
+    "top-level-and": ["node--segments-container"],
+    "noun": ["color2"],
+    "modifier": ["color3"],
+    "implied": ["placeholder"]
+  },
+  "linkToPosition": {
+    "subj": "left",
+    "obj": "right",
+    "parg": "right"
+  },
+  "linkNameToLabel": {
+    "subj": "S",
+    "obj": "O",
+    "parg": "O"
+  },
+  "root": {
+    "nodeType": "top-level-and",
+    "word": "and",
+    "link": "none",
+    "alternateParseInfo": {
+      "charNodeRoot": {
+        "charLo": 0,
+        "charHi": 85
+      },
+      "spanAnnotations": [
+        {
+          "spanType": "child",
+          "lo": 0,
+          "hi": 63
+        },
+        {
+          "spanType": "ignored",
+          "lo": 63,
+          "hi": 64
+        },
+        {
+          "spanType": "child",
+          "lo": 61,
+          "hi": 84
+        },
+        {
+          "spanType": "ignored",
+          "lo": 84,
+          "hi": 85
+        }
+      ]
+    },
```

Comment:
We shouldn't include a tree that's in the syntax the explainer produces, as people will use these examples to figure out how to create data that's compatible with the tool.  We don't want them to use this format, as it's hard to use and something we'll migrate away from in the longer term.

Refer to the examples I have in `hierplane/` as to figure out the protocol.  Also, I'll try to document that this week.

Reply:
Understood, and no problem. It wasn't immediately obvious to me how to convert the Explainer span syntax into Hierplane span syntax. Would love to sit down w/ you to discuss that. Am happy to make the change once I know how to do it. :)

Reply:
As discussed via Slack, we will probably postpone this item for the time being. An issue has been created to address this: https://github.com/allenai/hierplane/issues/59